### DISCUSSION TITLE
Single instance mode

### DISCUSSION THREAD

**Discussion by deathtrip** [neovide/neovide#2638] [Ideas]
When neovide is set as the default editor, it would be useful to have files opened from external applications be in the same window.
Maybe another option to select whether to open the new buffer in a tab or split.

**Comment by fredizzimo** [comment]
We have an issue open for this:
* https://github.com/neovide/neovide/issues/1586

And an example of how it can be done currently https://github.com/neovide/neovide/issues/1586#issuecomment-2118778451, not the most user-friendly way but it's doable.
